### PR TITLE
fix(learn): address translator comments

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Addresses some feedback provided by translators in Crowdin.

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/quoting-strings-with-single-quotes.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Right now, the `<a>` tag in the string uses double quotes everywhere. You will n
 
 # --hints--
 
-You should remove all the `backslashes` (<code>\\</code>).
+You should remove all the backslashes (<code>\\</code>).
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/quoting-strings-with-single-quotes.md:46]: I believe this now works fine on Crowdin now.
```suggestion
You should remove all the backslashes (`\`).
```

Comment (nhcarrigan) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/quoting-strings-with-single-quotes.md:46]: https://freecodecamp.crowdin.com/translate/690087755819f2f67b26dbee3be0eb53/21404/en-esem?filter=basic&value=13#q=725612

I tagged you here - this appears to be an instance where that bug occurs again.

Could it be that we need to re-upload the file to run the parser again?

Comment (RandellDawson) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/quoting-strings-with-single-quotes.md:46]: I think when you send them an email, you ask them why it is broken again.

**Review by RandellDawson — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**